src/main/steps/respondent/task-list/content.test.ts

```diff
@@ -54,18 +54,22 @@ const cyContent = {
     [SectionStatus.READY_TO_VIEW]: 'Yn barod i’w gweld',
     [SectionStatus.NOT_AVAILABLE_YET]: 'Ddim ar gael eto',
     [SectionStatus.DOWNLOAD]: 'LLWYTHO',
-    [SectionStatus.VIEW]: 'VIEW (in Welsh)',
+    [SectionStatus.VIEW]: 'GWELD',
   },
   sectionTitles: respondent_cy,
   taskListItems: respondent_tasklist_items_cy,
   iWantTo: 'Rwyf eisiau ...',
   hyperlinks: [
     {
-      label: 'Add a legal representative-welsh',
+      label: 'Ychwanegu cynrychiolydd cyfreithiol',
       link: '/respondent/add-legal-representative',
     },
     {
-      label: 'Find my local court-welsh',
+      label: 'Dileu cynrychiolydd cyfreithiol',
+      link: '/respondent/remove-legal-representative/start',
```

Comment:
Why is the link pointing to wrong one?

Reply:
As of the recent changes... They are pushing FInd my local court downwards and Add legal rep , Remove legal rep upwards.. so thats why its showing as a wrong link.